app/src/main/res/layout/color_dialog.xml

```diff
@@ -0,0 +1,275 @@
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:text="@string/letterBlue"
+                        android:textSize="20sp" />
+
+                    <com.example.drawingactivity.ForcableEditText
+                        android:id="@+id/blueText"
+                        style="@style/Widget.AppCompat.EditText"
+                        android:layout_width="50dp"
+                        android:layout_height="wrap_content"
+                        android:text="0"
+                        android:textSize="20sp" />
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:text="@string/letterGreen"
+                        android:textSize="20sp" />
+
+                    <com.example.drawingactivity.ForcableEditText
+                        android:id="@+id/greenText"
+                        style="@style/Widget.AppCompat.EditText"
+                        android:layout_width="50dp"
+                        android:layout_height="wrap_content"
+                        android:text="0"
+                        android:textSize="20sp" />
+
+                </LinearLayout>
+            </LinearLayout>
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_margin="5dp"
+            android:layout_weight="1"
+            android:gravity="center"
+            android:orientation="vertical">
+
+            <com.example.drawingactivity.ForcableEditText
+                android:id="@+id/hexText"
+                style="@style/Widget.AppCompat.EditText"
+                android:layout_width="100dp"
+                android:layout_height="wrap_content"
+                android:text="#ff0000"
+                android:textSize="20sp" />
+
+            <Button
+                android:id="@+id/okButton"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="OK" />
+
+            <Button
+                android:id="@+id/nopeButton"
```

Comment:
shouldn't this be something liek cancelButton? especialy if the text is cancel, and if you have to specify which sheet it would be like colorPickerCancelButton.

Reply:
I'll change the name on this one, it should match what I assign the variable for. Most Dialogs have a cancel button so it would make sense for this to follow that convention.